code/modules/events/anomaly_bluespace.dm

```diff
@@ -54,8 +54,8 @@
 					if(A.anchored) continue
 
 					var/turf/newloc = locate(A.x + x_distance, A.y + y_distance, TO.z) // calculate the new place
-					if(!A.Move(newloc)) // if the atom, for some reason, can't move, FORCE them to move! :) We try Move() first to invoke any movement-related checks the atom needs to perform after moving
-						A.loc = locate(A.x + x_distance, A.y + y_distance, TO.z)
+					if(!A.Move(newloc) && newloc) // if the atom, for some reason, can't move, FORCE them to move! :) We try Move() first to invoke any movement-related checks the atom needs to perform after moving
+						A.loc = newloc

```

Comment:
This was something I tried bringing up before I made this PR. As is stands, we have over 840 uses of the "X.loc = Y" syntax, and it strikes me that most of those should probably be X.forceMove(Y) instead. I have no problem with making 840 changes on a case by case basis, but if I were to do this, then it would have made the PR absolutely massive, and possibly conflict with a PR or two.